tests/tests/class-resizing.php

```diff
@@ -190,6 +190,7 @@ function data_filtered_url() {
 				'large',
 				[
 					'http://tachy.on/u/tachyon.jpg?fit=1024,575',
+					'http://tachy.on/u/tachyon.jpg?fit=1024,719',
```

Comment:
Didn't realise these test URLs existed; we should consider using tchyn.io, since we actually own it :D

Reply:
Ha! Peter’s been busy busy

Reply:
Unit tests are dandy 🌻 

Although I am disappointed the test images are of a Tachometer rather than Tachyon, the K-pop boy band.

Tahanks for fixing it all, @roborourke, sorry I broke your stuff.

Reply:
Nah it's cool @peterwilsoncc - you were definitely right about some of the stuff I'd done wrong previously. It's all so very confusing